### PR TITLE
Fix/77452 improve ea settings wording

### PR DESCRIPTION
**Pull request by niconerd**
https://central.tri.be/issues/77452

### CODE REVIEW

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:x: | String &quot;Fetch source events settings (e.g. Show Google Maps Link or Sticky in Month View) when importing from another site using The Events Calendar.&quot; does not require double quotes; use single quotes instead | [settings.php:411](https://github.com/moderntribe/the-events-calendar/blob/5e6b091c8a799d42f8aaf10d04a62e4a7a99acd5/src/admin-views/aggregator/settings.php#L411)

> via jenkins/codesniffer-phpcs

File: src/admin-views/aggregator/settings.php [hunk 1]
```diff
@@ -408,7 +408,7 @@
 		'tribe_aggregator_default_url_import_event_settings' => array(
 			'type' => 'dropdown',
 			'label' => esc_html__( 'Import Event Settings', 'the-events-calendar' ),
-			'tooltip' => esc_html__( "Fetch source event's settings (e.g. Show Google Maps Link or Sticky in Month View) when importing from another site using The Events Calendar.", 'the-events-calendar' ),
+			'tooltip' => esc_html__( 'Fetch source events settings (e.g. Show Google Maps Link or Sticky in Month View) when importing from another site using The Events Calendar.', 'the-events-calendar' ),
```
Comment (barryhughes) [inline src/admin-views/aggregator/settings.php]: I checked with our team grammarian and we feel the apostrophe is appropriate here, given the context, so rolling back this change would be ideal.

Comment (barryhughes) [reply src/admin-views/aggregator/settings.php]: ...but keep the other change

Comment (niconerd) [reply src/admin-views/aggregator/settings.php]: ready for another review

**Pull request merged**